doc/user-guide.md

```diff
@@ -156,7 +156,15 @@ return reconcile.Result{}, err
 return reconcile.Result{Requeue: true}, nil
 ```
 
-**// TODO:** Doc on controller-runtime client and examples on how to use it.
+You can set the `Resulte.RequeueAfter` to requeue the `Request` after a grace period as well:
+```Go
+import "time"
+
+// Reconcile for any reason than error after 5 seconds
+return reconcile.Result{RequeueAfter: time.Duration(5)}, nil
```

Comment:
nit:
```suggestion
return reconcile.Result{RequeueAfter: time.Second*5}, nil
```

 I prefer this way because it more clearly shows the unit IMO

Reply:
I think `time.Duration(5)` would be 5 nanoseconds as well, not 5 seconds